src/builder/commands/alpine.rs

```diff
@@ -21,6 +21,9 @@ static ALPINE_VERSION_REGEX: &'static str = r"^v\d+.\d+$";
 static MIRRORS: &'static str = include_str!("../../../alpine/MIRRORS.txt");
 
 
+const VERSION_WITH_PHP5: f32 = 3.4;
```

Comment:
Well, keeping version as floating point is definitely wrong.

Either use a tuple `(3, 4)` or a string (`v3.4`), you can wrap latter to `config::version::Version` for correct version comparison (probably the wrapper might be moved out of config module)